wolfhsm/wh_server.h

```diff
@@ -75,15 +75,15 @@ typedef struct {
 #ifdef WOLFSSL_CMAC
         Cmac cmac[1];
 #endif
-    };
+    } crypto_context_crypto;
     union {
```

Comment:
I think you might need to move this up to `union crypto_context_pubkey {`. Otherwise you'll have to change the calling code.

Reply:
For reference... Doxygen will find the "name" is either place, but if you change it after then you'll have to change the calling code to use `. crypto_context_crypto`.

Reply:
yep, you are correct just saw that. Unfortunately the way you originally had it generates a compiler warning, which I was trying to get rid of. But realize now that the way I did it is invalid. Hmmm....

```
wh_server.h:86:6: warning: declaration does not declare anything
   86 |     };
```

where the `};` reference is the closing bracket of the nested union.